scribejava-apis/src/main/java/com/github/scribejava/apis/service/GeniusOAuthServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package com.github.scribejava.apis.service;
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.model.AbstractRequest;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+import com.github.scribejava.core.model.OAuthConfig;
+import com.github.scribejava.core.model.OAuthConstants;
+import com.github.scribejava.core.oauth.OAuth20Service;
+
+public class GeniusOAuthServiceImpl extends OAuth20Service {
+	private static final String USER_AGENT = "user-agent";
+	private static final String AGENT_NAME = "example_name";
```

Comment:
Maybe change it to something more appropriate. Yes, we need it - I was getting authorisation errors, apparently this is a security feature against bots (http://stackoverflow.com/questions/24913699/the-owner-of-this-website-has-banned-your-access-based-on-your-browsers-signat).

 Consider adding this as a configurable option? (I can do that if you want)